Jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
       parallel {
         stage('Integration Tests') {
           steps {
-            sh 'summon ./test.sh'
+            sh 'summon ./ci/test_e2e'
```

Comment:
We'll need to split this into two steps, one that calls `summon ./ci/test_e2e` and one that calls `./bin/test_integration`.